pkg/detectors/falsepositives.go

```diff
@@ -12,7 +12,10 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
 )
 
-var DefaultFalsePositives = []FalsePositive{"example", "xxxxxx", "aaaaaa", "abcde", "00000", "sample", "*****"}
+var (
+	DefaultFalsePositives = []FalsePositive{"example", "xxxxxx", "aaaaaa", "abcde", "00000", "sample", "*****"}
+	UuidFalsePositives    []FalsePositive
```

Comment:
Having a list for UUIDs is useful if you have a value that's supposed to be a UUID.